### DISCUSSION TITLE
How deleteAll items with `relation.count` = 0

### DISCUSSION THREAD

**Discussion by cawa-93** [prisma/prisma#13559] [Q&A] [answered]
Hello. There are `templates` and `links` models in my database that are related as 1-n.
I want to delete all `templates` that does/t have related links
```ts
db.template.deleteMany({
  where: {
    label: null,
    links: {
      /* ??? */
    }
  }
})
```

**Comment by cawa-93** [comment, marked as answer]
My bad. Found: https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#filter-on-presence-of-related-records

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
